app/assets/stylesheets/functions/_strip-units.scss

```diff
@@ -14,8 +14,8 @@
 
 @function strip-units($value) {
   @if $output-bourbon-deprecation-warnings == true {
-    @warn "[Bourbon] [Deprecation] `strip-units` is deprecated and will be " +
```

Comment:
This function is deprecated, so I think we should leave in `[Deprecation]` and `is deprecated`.

According to [Wikipedia](https://en.wikipedia.org/wiki/Deprecation#Software_deprecation):

> …deprecated status may also indicate the feature will be removed in the future.

As you mentioned in #1013, “Use the renamed `strip-unit` function instead” is a bit confusing because you can’t use that _yet_ if you are still on 4.x. So we should re-word only that sentence to make it clear that you will need to use the renamed function when you upgrade to 5.0.